### PR TITLE
(tixati) Overhauled AHK script for v2 (attempt 2)

### PR DESCRIPTION
**Pull request by teknowledgist**
## Description
Rewrote the AutoHotKey script to be compatible with AHKv2.

## Motivation and Context
The Tixoti package has an "open" dependency on AutoHotKey. When V2 of AHK was realeased, it had significantly different syntax, and it broke this package (and many others). The AHK script had to be rewritten. Also, launching the installer from the ChocolateyInstall.ps1 script rather than from AHK seems more "proper".

## How Has this Been Tested?
`choco pack` and `choco install tixoti -s . -y` and `choco uninstall tixoti -y`

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:
- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] I have updated the package description and it is less than 4000 characters.
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Review by pauby — APPROVED** [review]

**Comment by pauby** [comment]
@AdmiringWorm @JPRuskin @Windos @gep13 @corbob (I'm missing somebody):

image

I've tested this PR in the Chocolatey Test Environment, and it installed and uninstalled correctly. It also output the correct text.

I've fixed up this PR, but cannot merge. Could somebody do that?

@AdmiringWorm we should look to amend this. If I fix a PR up, I want to be able to merge it. If I submit a PR, I should not be able to merge it.

@teknowledgist Thanks for submitting this and apologies it took so long to review. I've fixed a few things up, and it's pending a merge. I've added back in the tags that were removed as we need to ensure we have a thumbs up from the maintainers before doing that.

**Comment by pauby** [comment]
@TheCakeIsNaOH I didn't know you had merge permission here, or I would have tagged you. Thank you for merging it.

**Comment by AdmiringWorm** [comment]
@pauby I have disabled the rule about requiring approval of the latest pushes (this is the one that prevented you from merging this pull request).

I have put it up for discussion as part of our next scheduled meeting whether we want to have this enabled or disabled.

**Pull request merged**
